src/index.js

```diff
@@ -0,0 +1,108 @@
+import './scss/battleship.scss';
+import UI from './gameui';
+import GameBoard from './gameboard';
+import AI from './ai';
+
+let gameOver = false;
+UI.generateBoard();
+
+// Place ships on waters
```

Comment:
**Kindly remove commented code, you can always use git history to access previous code right :slightly_smiling_face:**
Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Reply:
Thank you @bellom so far on my code reviews. I appreciate your prompt responses. 
Correct me if I am wrong, but I was thinking that this reference to deleting a comment is for only ``commented code`` probably, previously working/non-working code, and not entirely for minor descriptive inline comments like ``Game Loop``, ``Change Player Turn``, ``Place ships on waters`` etc.

However, I have removed the comments as you suggested.